client/src/components/CurseHome.tsx

```diff
@@ -0,0 +1,190 @@
+        break;
+    case 'FEMININ2':
+        setStartTime('Ora de start: 14:10');
+        setDistance('DISTANȚĂ: 2KM');
+        setMinimumDonation('Donatie minima: 40RON');
+        setMapSrc('https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2848.878166115072!2d26.051764615851956!3d44.43970637910117!2m3!1f0!2f0!3f0!3m2!1d44.43998527916441!2d26.051764615851956!4f0!5f0.7820865974627469');
+        break;
+    case 'MASCULIN2':
+        setStartTime('Ora de start: 15:00');
+        setDistance('DISTANȚĂ: 4KM (2 x 2KM)');
+        setMinimumDonation('Donatie minima: 40RON');
+        setMapSrc('https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2848.878166115072!2d26.051764615851956!3d44.43970637910117!2m3!1f0!2f0!3f0!3m2!1d44.43998527916441!2d26.051764615851956!4f0!5f0.7820865974627469');
+        break;                
+      default:
+        break;
+    }
+  };
+
+  return (
+    <div className='w-full px-[var(--bs-gutter-x,.75rem)]   bg-customBackground  '>
+        {/* <img src={flyingImg4} className='float-right top-0 right-0 h-full md:w-1/2 lg:w-1/3 xl:w-1/4 z-0' alt='Imagine de fundal' ></img>  */}
+    <div className='flex justify-center '>    
+    <div className="container  md:pt-32  lg:px-32 mx-auto  " >
+        <h1 className="lg:text-52 text-36 font-bold text-center mb-8 sm:mb-16 text-black">CURSE</h1>
+        <div className="md:flex">
+            
+                <div className="md:w-1/4 mb-4 md:mb-0 md:mr-4 overflow-x-auto">
+                    <div className="flex sm:flex-row md:flex-col">
+                        <button
+                        className="bg-[#006470] hover:bg-teal-500  rounded-md md:text-right sm:text-center lg:px-35 lg:py-33 md:px-20 md:py-20 px-20 py-8 "
+                        onClick={() => handleCategoryChange('All for one')}
+                        >
+                            <span className="font-bold lg:text-24 md:text-20 sm:text-20 text-white  whitespace-nowrap">ALL FOR ONE</span>
+                            <br/>
+                            <span className="lg:text-16 md:text-14 text-white ">TOATE VÂRSTELE</span>
+                        </button>
+                        <button
+                        className="bg-[#006470] hover:bg-teal-500  rounded-md md:text-right sm:text-center lg:px-35 lg:py-33 md:px-20 md:py-20 px-20 py-8"
+                        onClick={() => handleCategoryChange('Copii')}
+                        >
+                        <span className="font-bold lg:text-24 md:text-20 sm:text-20 text-white  whitespace-nowrap">COPII</span>
+                            <br/>
+                        <span className="lg:text-16 md:text-14 text-white whitespace-nowrap">&lt; 10 ANI</span>
+                        </button>
+                        <button
```

Comment:
All of these have the same structure, so there should be just one where you map based on what you have in the JSON. It's unnecessary to change the characteristics of the component like that.